base/applications/kbswitch/lang/pt-PT.rc

```diff
@@ -0,0 +1,18 @@
+/*
+ * PROJECT:     Keyboard Layout Switcher
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Portuguese (Portugal) resource file
+ * TRANSLATOR:  Copyright 2024 Jose Carlos Jesus <zecarlos1957@hotmail.com>
+ */
+
+LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL
+
+IDR_POPUP MENU
+BEGIN
+    POPUP "popup"
+    BEGIN
+        MENUITEM "&Preferências...", ID_PREFERENCES
+        MENUITEM SEPARATOR
+        MENUITEM "&Fechar a barra de idiomas", ID_EXIT
```

Comment:
Note that en-US text is just "Exit".

Reply:
This menu item complies with Win2003 Srv "&Fechar a barra de idiomas"

Reply:
Win2003 has no kbswitch.exe, it's ReactOS own component. See also https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=ID_EXIT